ui/panels-plugin/src/plugins/time-series-table/TimeSeriesTable.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { PanelPlugin, PanelProps } from '@perses-dev/plugin-system';
+import { TimeSeriesTablePanel } from './TimeSeriesTablePanel';
+
+interface TimeSeriesTableOptions {}
+
+/**
+ * The core TimeSeriesTable panel plugin for Perses.
+ */
+export const TimeSeriesTable: PanelPlugin<TimeSeriesTableOptions> = {
```

Comment:
I need to find a way to set by default the TimeSeriesTable

Also, in other PRs, we can support the instant/range switch for other Charts like TimeSeriesChart, GaugeChart etc...

Reply:
First part solved introducing the queryOptions field here.

Reply:
Second part is more complex as we would need then to save the switch selection in query specs

Reply:
I don't think it's a good idea to mixed panel in the same plugin.

What would be the benefit of doing that ? Excepting in an explorer mode, I don't see why in a dashboard you would need to switch from a range query to an instant query.

Reply:
This might be not useful in TimeSeriesChart yes, or at least I don't have an idea of the use case. But when you do use a gauge chart for example. You may want to have the choice between using a range query and display the average/last/min etc or make an instant query that would be more performant if you know you want the last.